### PR TITLE
Frontend Basic Design

### PR DESCRIPTION
**Pull request by ngblaylock**
We may need to use the fullscreen API, but I would like to see where we are at with just this. 

https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+              <td class="penalty-time">1:47</td>
+            </tr>
+            <tr>
+              <td class="penalty-player">62</td>
+              <td class="penalty-time">:55</td>
+            </tr>
+          </table>
+        </div>
+      </div>
+      <div class="advertisement">
```
Comment (jak103) [inline frontend/index.html:80]: Let's remove the advertisement. We don't need that. That was just that image I grabbed.

**Comment by jak103** [comment]
This will need to be connected to a web socket. We will also eventually want to add a second screen with a control panel that the score keepers can use to run animations and maybe sounds. 

We can turn this into a full web app and not just a static page.

File: frontend/index.html (hunk 1 above)
Comment (ngblaylock) [reply frontend/index.html:80]: Oh good. I wondered about that.

**Comment by ngblaylock** [comment]
> This will need to be connected to a web socket. We will also eventually want to add a second screen with a control panel that the score keepers can use to run animations and maybe sounds.
> 
> We can turn this into a full web app and not just a static page.

That is good to know. I was wondering if we needed a build setup or if we could get away with something like petite vue.

**Pull request merged**